src/js/index.js

```diff
@@ -0,0 +1,65 @@
+import operate from './modules/operator.js'
+
+let number = ''
+let operator = ''
+let numberList = []
+
+function caculator() {
+    clickNumber()
+    clickOperator()
+    modifier()
+}
+caculator()
+
+function clickNumber() {
+    document.querySelectorAll('.digit').forEach(function(btn) {
+        btn.addEventListener('click', function() {
+
+            if(operator === '=') {
+                initialize()
+            }
+
+            if(number.split('').length <= 2) {
+                if(!(number === '' && btn.innerText === '0')) {
+                    number = number.concat(btn.innerText)
+                    document.getElementById('total').innerText = number
+                }
```

Comment:
```suggestion
if(number === '' && btn.innerText === '0') {
    return;
}
                
number = number.concat(btn.innerText)
document.getElementById('total').innerText = number
```

validation같은 경우는 early return을 하는게 훨씬 가독성에 좋답니답